frontend/src/components/cluster-dashboard/ClusterDashboard.tsx

```diff
@@ -44,12 +45,16 @@ const ClusterDashboard: React.FC = () => {
 
   return (
     <React.Fragment>
-      <Grid container md item direction="row" className="shukra-body">
+      <Grid container md
```

Comment:
More of a personal preference, but if we're breaking up lines for component elements, I like to completely split each prop/attribute onto a separate line so it's still easy to quickly parse what is being parsed. Otherwise, why split between md and item? It almost feels like the `item-direction="row"` which isn't the case. `item={true}` and `direction="row"` here, but it's a little confusing with them being on the same line.